### DISCUSSION TITLE
False detections on VT?

### DISCUSSION THREAD

**Discussion by ghost** [Tyrrrz/DiscordChatExporter#1013] [Q&A] [answered]
Hi,

First of all, this program worked great so **thank you**.

I fully believe that this project is not some long con to steal discord tokens and has proper security practices but I can't help but ask if the (currently four) VT detections (see here: https://www.virustotal.com/gui/file/1041117a6b278df3f07ccf9599397dade008adb320ec3da2d864f570a585aa31) are just false positives or something to worry about?

Any ideas why the program may get flagged? Was that also the case for previous releases?

_Thanks!_

**Comment by Technetium1** [comment, marked as answer]
They're generic detections by some slightly obscure engines, nothing to really worry about with those. Sometimes a single line of code formatted a certain way can be enough to trigger a detection.
